src/config/baseline.py

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
It's minor, but still -- we can delete this empty line

Reply:
of course